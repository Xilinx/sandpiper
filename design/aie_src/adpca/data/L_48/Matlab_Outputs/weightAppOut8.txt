3.378614426e+00 3.452127934e+00
3.098939896e+00 1.202252388e+00
7.083927155e+00 -4.387001514e+00
