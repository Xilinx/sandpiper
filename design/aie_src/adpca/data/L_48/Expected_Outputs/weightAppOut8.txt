3.378491402e+00 3.452083349e+00 
3.098810434e+00 1.202338696e+00 
7.083687782e+00 -4.386901855e+00 
